styles/styles.css

```diff
@@ -0,0 +1,100 @@
+/* Footer */
+
+footer {
+    background-color: white;
+}
+
+.footer-container {
+    width: 100%;
+    padding: 70px 0px 20px;
+}
+
+.brand-logo {
+    width: 200px;
+}
+
+.logo-div {
+    display: flex;
+    justify-content: center;
+}
+
+.social-icons {
+    display: flex;
+    justify-content: center;
+}
+
+.social-icons a {
+    text-decoration: none;
+    padding: 10px;
+    background-color: white;
+    margin: 10px;
+    border-radius: 50%;
+}
+
+.social-icons a i {
+    font-size: 1.5rem;
+    color: black;
+    opacity: .8;
+}
+
+.social-icons a:hover {
+    background-color: #333;
+    transition: .5;
+}
+
+.social-icons a:hover i {
+    color: white;
+    transition: .5;
+}
+
+.footer-nav {
+    margin: 30px 0;
+}
+
+.footer-nav ul, li {
```

Comment:
` i` could target `<li>,</li>` globally. You can put it up to the top of the file instead or in `.footer-nav ul li`